Tasks/Kudzelich/Task1/1_1.py

```diff
@@ -0,0 +1,10 @@
+C = input("Enter your value in Celsius:")
+if C.isnumeric():
+    C=float(C)
+    F = C*1.8+32
+    print(f"{C} in Celsius is {F} in Fahrenheit")
+else:
+    C = input("Please, enter value in numeric format:")
+    C=float(C)
+    F = C*1.8+32
+    print(f"{C} in Celsius is {F} in Fahrenheit")
```

Comment:
code copy-paste is a huge mistake, but do not worry about it for now, I will show how to avoid it

Reply:
Ok, thank you :)